src/openrct2/management/Marketing.cpp

```diff
@@ -237,3 +237,20 @@ void marketing_new_campaign(const MarketingCampaign& campaign)
         gMarketingCampaigns.push_back(campaign);
     }
 }
+
+void marketing_cancel_campaigns_for_ride(ride_id_t rideId)
```

Comment:
All new functions should be TitleCase. rideId should be const

Reply:
I'ma bit torn between using and not using const for values passed by value. It clusters the code if used a lot and in some cases it makes it look like a value is const while it only makes the pointer const (`char* const c`), though it helps to avoid side-effects (accidentally using `=` instead of `==`).

I'd say we only do this in source files, the header file doesn't care whether or not the value gets changed.

Reply:
Adding more consts reduces errors and conveys more information. You can put it in both the header and the source file. I'll admit that some instances of very short functions it is a bit annoying to const the variables though this isn't one of those situations